dom/events/Event-initEvent.html

```diff
@@ -88,7 +88,6 @@
     assert_unreached("")
   }), false)
   assert_equals(target.dispatchEvent(e), true, "dispatchEvent must return true")
-  assert_equals(target.dispatchEvent(e), true, "dispatchEvent must return true")
```

Comment:
Actually, this entire test now no longer tests what it's supposed to.  It's checking if the stop propagation flag is unset by initEvent(), but calls dispatchEvent() in between, so the flag will be reset anyway by that.  It needs to be rewritten.  It happens to be I wrote some code that does that anyway, so I'll handle that in a separate PR.